defaults/main.yml

```diff
@@ -4,3 +4,7 @@ deploy_group: '{{ ansible_ssh_user }}'
 
 laravel_rotate_nginx_log: true
 laravel_rotate_nginx_log_retention: 30
+
+laravel_https_enabled: 'on'
```

Comment:
Let's make this variable a boolean (yes/no) so adding certificate files and using https is optional. Not all applications will need it, but should still be able to use this role.